src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -129,7 +129,7 @@ public function getEnv($prefix, $name, \Closure $getEnv)
                 throw new RuntimeException(sprintf('Invalid JSON in env var "%s": '.json_last_error_msg(), $name));
             }
 
-            if (!is_array($env)) {
+            if (null !== $env && !is_array($env)) {
                 throw new RuntimeException(sprintf('Invalid JSON env var "%s": array expected, %s given.', $name, gettype($env)));
```

Comment:
The error message should be updated: `array expected ...` -> `array or null expected ...`